tutorials/Land/Heat/heat_equation.jl

```diff
@@ -417,12 +417,13 @@ time_data = FT[0]                                      # store time data
 
 export_plot(
     z,
+    time_data,
     all_data,
     ("ρcT",),
     joinpath(output_dir, "initial_condition.png");
     xlabel = "ρcT",
     ylabel = z_label,
-    time_data = time_data,
+    xlims = (m.initialT - 1, m.initialT + 1),
```

Comment:
`T_bottom` is 300 K and `initialT` is 295 K, I would set `xlims = (m.initialT - 1, m.T_bottom + 1)`